subsys/bluetooth/Kconfig

```diff
@@ -83,6 +83,42 @@ config BT_BROADCASTER
 
 endmenu
 
+config BT_ADV_EXT
+	bool "Extended Advertising and Scanning support"
+	help
+	  Select this to enable Extended advertising API support.
+	  This enable multiple advertising sets with extended advertising
+	  data and advertising with long range suppert.
+	  Enables extended scanner with support for receiving extended
+	  advertising data and scanning with long support.
+	  Enables the ability create connection using the long range feature.
+
+if BT_ADV_EXT
+config BT_ADV_EXT_LEGACY_SUPPORT
+	bool "Support starting advertising through legacy commands"
+	help
+	  Select this to enable the use of the Legacy Advertising HCI commands.
+	  This option should be used where the capabalities of the controller
+	  is not known.
+
+config BT_ADV_EXT_LEGACY_ONLY
```

Comment:
Why is this needed? Won't all controllers that support AE support the legacy commands too?

Reply:
This was intended as an option for when using the new advertising set API was not needed, but using the new extended
HCI commands was needed. It doesn't seem  like there is much to gain from this option in terms of code reduction though. So I think I will just remove this option.

I think what we might need instead however is to assume the controller supports the feature, and skip checking the feature flag. For combined builds this would make sense, and we could compile out all the legacy HCI commands.

Reply:
>I think what we might need instead however is to assume the controller supports the feature, and skip checking the feature flag. For combined builds this would make sense, and we could compile out all the legacy HCI commands.

Agreed, for combined builds in the future (when we support AE in the controller) then `BT_ADV_EXT_LEGACY_SUPPORT` should default to `n` (once it's implemented).